src/devices/tpms_bmw.c

```diff
@@ -0,0 +1,150 @@
+/** @file
+    BMW Gen5 TPMS sensor.
+    Copyright (C) 2024 Bruno OCTAU (ProfBoc75), @petrjac, Christian W. Zuckschwerdt <christian@zuckschwerdt.org>
```

Comment:
One empty line above needed. The `@petrjac` needs to be `\@petrjac` (this needs to be Markdown and also valid Doxygen)